File: codigos/media.java
import java.io.*;
public class Media {
    public static void main(String[] args) {
        BufferedReader  inReader;
        inReader = new BufferedReader(new InputStreamReader(System.in));
        String line;
        int numero1, numero2 = 0;
        double media = 0;
        try {
            System.out.print("Digite o primeiro número: ");
            line = inReader.readLine();
            numero1 = Integer.parseInt(line);
            System.out.print("Digite o segundo número: ");
            line = inReader.readLine();
            numero2 = Integer.parseInt(line);
            media = (numero1 + numero2) / 2.0;
            System.out.println("A média é " + media);
            inReader.close();
        } catch (IOException e) {
            System.err.println(e.getMessage());
        }
    }
}

File: codigos/media.cpp
#include <iostream>
using namespace std;
int main() {
	double numero1, numero2, media;
	std::cout << "Digite o primeiro número: ";
	std::cin >> numero1;
	std::cout << "Digite o segundo número: ";
	std::cin >> numero2;
	media = (numero1 + numero2) / 2.0;
	std::cout << "A média é " << media << endl;	 
	return 0;
}
